esp/src/src/ESPRequest.ts

```diff
@@ -65,14 +70,14 @@ class RequestHelper {
         }
         // var method = params.method ? params.method : "get";
 
-        var retVal = null;
-        if (this.isCrossSite()) {
-            var transport = new hpccComms.Connection({ baseUrl: this.getBaseURL(service), type: "jsonp" });
-            retVal = transport.send(action + postfix, params.request, handleAs === "text" ? "text" : "json");
-        } else {
-            var transport = new hpccComms.Connection({ baseUrl: this.getBaseURL(service) });
-            retVal = transport.send(action + postfix, params.request, handleAs === "text" ? "text" : "json");
+        if (this.isLocked()) {
+            throw new Error("session locked");
         }
+
+        var retVal = null;
+        var transport = new hpccComms.Connection({ baseUrl: this.getBaseURL(service) });
+        retVal = transport.send(action + postfix, params.request, handleAs === "text" ? "text" : "json");
```

Comment:
Are we dropping the support for "isCrossSite"?  It may not be working now anyway (this was where we opened ECL Watch on 160 and pointed it to dataland).

Reply:
I removed because I thought we were dropping the support for this? I can leave as is?